packages/e2e-tests/tests/cypress-03_test-step-interactions.test.ts

```diff
@@ -153,6 +153,8 @@ test("cypress-03: Test Step interactions", async ({ pageWithMeta: { page, record
   });
 
   await steps.nth(9).click();
+  await delay(100);
```

Comment:
💬 I'm a bit surprised that a delay would help here.  Either I'd expect the other step clicks to have a similar issue, or that this would be okay.

Also, from the triage, it looked like it was having trouble finding a `KeyValue-Header` for "Selected" even though that appeared to be visible on screen.

If this actually does fix things, great, I'm all for that :) but based on what I saw I wouldn't expect this to resolve the failures.

Reply:
Pretty sure it was still reading DOM for the the wrong/previous thing. Reliably failed before for me and reliably passed after this change.

Reply:
Ah, okay, that makes more sense! Clicking, but the _old_ step details are still there and there's a mismatch.